libunwind/src/DwarfInstructions.hpp

```diff
@@ -54,6 +55,15 @@ class DwarfInstructions {
   typedef typename CFI_Parser<A>::FDE_Info          FDE_Info;
   typedef typename CFI_Parser<A>::CIE_Info          CIE_Info;
 
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(_LIBUNWIND_SANDBOX_OTYPES)
```

Comment:
Can we just use the libunwind native types here?  That is, just use `pint_t` instead of `uintptr_t` and probably use it instead of `uintcap_t` as well?  In particular, in purecap code, we don't need to use `uintcap_t` and can instead just use `uintptr_t` normally.  Typically `uintcap_t` is only used in hybrid code and all of this is purecap.

Reply:
As discussed, `pint_t` is now used everywhere except for `Registers.hpp` which can't include `AddressSpace.hpp`.